client/src/CONSTANTS.ts

```diff
@@ -0,0 +1,32 @@
+export class CONSTANTS {
```

Comment:
I prefer only things that are used more than once are abstracted to constants. Otherwise, I find it easier to have the values inlined